README.md

```diff
@@ -0,0 +1,213 @@
+
+<p align="center">
+<img  align="center" src="https://raw.githubusercontent.com/gemini-hlsw/gmp/igrins2/images/SimpleGMPArhictecture.jpg">
+</p>
+
+# Desployment from binaries. 
+It is possible deploying the GMP and GIAPI-GLUEc++ from their binaries. Gemini recommend uses Rocky 8 or Centos 7 because these are the Operating Systems supported by GEMINI and for which the GIAPI-GLUEc++ binaries have been compiled. 
+
+If you want to deploy the system in other platform, you should compile the external libraries of the [GIAPI-GLUEc++](https://github.com/gemini-hlsw/giapi-glue-cc/tree/develop/external) for you platform. In the previous link is decribed the instrucction executed by GEMINI to CENTOS7 and ROCKY8.
+
+## Dependencies 
+The only dependencies we need for GMP and GIAPI-glue are as follows.
+* Java JDK 1.8
+* GCC greater than 4.8.1.
+
+If you are on Rocky 8, you could use the following commands:
+* dnf -y install java-1.8.0-openjdk-devel
+* dnf -y group install "Development Tools"
+
+## Deployment instructions. 
+This section describes the instructions for deploying the GMP and [any-command](https://github.com/gemini-hlsw/giapi-glue-cc/tree/develop/src/examples/any-command.cpp) example implemented in GIAPI-GLUE. The any-command is an example created by GEMINI where developers can see how to implement a command subscriber using the GIAPI-GLUEcc library. Then, we are going to use the giapi-tester to send a command to the any-command example, emulating how any client of the system could send a command to a subscriber. There are more examples in the [src/examples](https://github.com/gemini-hlsw/giapi-glue-cc/tree/develop/src/examples) folder of the GIAPI-GLUEcc repository. 
+
+For this example, the GMP and the any-command example will be deployed on the same machine. If you need deploy them on different machines, you have to modify the gmp.properties which are located in the src/examples folder. 
+
+
+### Deployment GMP
+Open a terminal or console to execute the following commands. 
+```
+> tar -xzvf gmp-server-igrins2-v0.2.0.tar.gz
+> cd gmp-server-0.2.0
+> ./bin/gmp-server-ctl.sh start
+```
+
+### Deployment any-command (GIAPI-GLUE example)
+Open a terminal or console to execute the following commands. 
+```
+> tar -xzvf giapi-gluecc_rocky8.tar.gz
+> cd giapi-gluecc
+> source defineGiapiglueEnv.sh
+> cd src/examples
+> If you deployed the GMP on other server or machine, you have to edit the gmp.properties file and specifying the IP or hostname of the server where the GMP was deployed (modify the gmp.hostname field)
+> ./any-command
+```
+
+### GIAPI-TESTER
+Open a terminal or console to execute the following commands. 
+```
+> cd gmp-server-0.2.0/bin
+> java -jar giapi-tester.jar -sc OBSERVE -activity START -config DATA_LABEL=S11172022S001.fits
```

Comment:
I suggest we use something else than OBSERVE which is often different, perhaps an apply?

Reply:
Ok. I added the apply command too.